client/src/components/Workflow/Editor/Attributes.vue

```diff
@@ -80,7 +79,7 @@ export default {
         },
         name: {
             type: String,
-            required: true,
```

Comment:
Is it actually possible to have an attribute with an empty/null name?

Reply:
Yes, I noticed a console log error regarding this attribute. I have not dug much deeper yet, but it seems that this attribute is not available when the Attributes component is generated and then populated at a later time through an async request. Removing the required attribute avoids the console error.

Reply:
That smells like a bigger problem -- if it's a prop here it should be available to the component from the start, not optional.

Reply:
Well, yea I agree we should look more into it. The goal of this commit here is to avoid the console error, although it's not really part of this PR. Maybe we should create a separate issue for it and follow-up. It could make it into the release as bug fix.